conf/messages

```diff
@@ -104,6 +104,7 @@ iht.delete = Delete
 iht.finishAndDeleteThisEstateReport = Finish and delete this estate report
 iht.the.deceased = the deceased
 iht.ifYouWantToChangeYOurAnswer = If you want to change your answer
```

Comment:
iht.ifYouWantToChangeYOurAnswer has been renamed to  iht.ifYouWantToChangeYourAnswer